unittests/linuxliketests.py

```diff
@@ -1920,3 +1920,7 @@ def test_persp_options(self):
         self.check_has_flag(compdb, mainsrc, '-O3')
         self.check_has_flag(compdb, sub1src, '-O2')
         self.check_has_flag(compdb, sub2src, '-O2')
+
+    def test_run_command_external_program(self):
```

Comment:
I think the suggestion was to add nativefile.ini to an existing test that uses run_command already.

That would allow avoiding the overhead of doing multiple project setups.